packages/app/src/components/ModeSelector.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <div class="container centered-container" justify-around style="margin-top: 20px;">
+      <div style="max-width:140px;" class="container column-container centered-container">
+        <q-btn
+          v-if="isTimedMode"
+          @click="setTimedMode"
+          round
+          size="40px"
+          color="primary"
+          icon="alarm"
+        />
+        <q-btn v-else @click="setTimedMode" round size="40px" icon="alarm" />
+        <p style="text-align:center;">How long would you like to play for?</p>
+      </div>
+      <h3>OR</h3>
+      <div style="max-width:140px;" class="container column-container centered-container">
+        <q-btn
+          v-if="isQuestionsMode"
+          @click="setQuestionsMode"
+          round
+          size="40px"
+          color="primary"
+          icon="help_outline"
+        />
+        <q-btn v-else @click="setQuestionsMode" round size="40px" icon="help_outline" />
+        <p style="text-align:center;">How many question would you like to play?</p>
+      </div>
+    </div>
+    <div class="conatiner" v-if="isTimedMode">
+      <p>How long would you like to practice for?</p>
+      <q-select
+        filled
+        v-model="selectedTime"
+        :options="timeOptions"
+        label="Choose Time"
+        style="width: 250px"
```

Comment:
Since this and the other select have a common style, you should move them to a class